Python/web_scraping_in_python/README.md

```diff
@@ -0,0 +1,44 @@
+I've completed the documentation of web scraping in Python that consists of:
+<ul>
+  <li>
+  <b>What is Web Scraping?</b><br/><br/>
+  Web scraping is defined as an automated system used to collect large quantities of data from other websites. 
+  The data on the websites are unshaped. Web scraping helps collect these unshaped data and store it in a structured form. 
+  There are different ways to scrape websites similar as online Services, APIs or writing your own code. In this documentaiton, I've covered Web Scraping with Python. 
+ 
+  </li>
+  <li>
+  <b>Why Python for Web Scraping?</b><br/><br/>
+  Python is a well-known device for executing web scraping. Python programming language is additionally utilized for other valuable activities 
+  identified with network safety, entrance testing just as advanced measurable applications. 
+  Utilizing the base programming of Python, one can start web scraping. Let's us see how Python is more preferable for Web Scraping:<br/>
+<br/>
+<ul>
+  <li><b>Punctuation Simplicity</b>
+Python has the most straightforward construction as compared with other programming languages. 
+    This element of Python makes the testing simpler and faster.
+  </li>
+  <li><b>Inbuilt Modules</b>
+Another reason for utilizing Python for web scraping is the inbuilt just as outside valuable libraries it has.
+    We can perform numerous executions identified with web scraping by utilizing Python as the base for programming.
+  </li>
+  <li><b>Open Source Programming Language</b>
+Python has tremendous help from the community since it is an open-source programming language.
+  </li>
+  <li><b>Wide scope of Applications</b>
+Python can be utilized for different programming assignments going from little shell contents to big business web applications.</li>
+<br/><br/></li>
+ </ul>
+ 
+ <li><b>Python Libraries Used For Web Scraping</b><br/><br/>
```

Comment:
mention and explain other libraries as well that are used for web scraping

Reply:
@apeksha235  I've made all the changes that you specified. Please have a look

Reply:
@apeksha235 @adityabisoi please review